tests/Aspire.Dashboard.Tests/Integration/Playwright/PlaywrightTestsBase.cs

```diff
@@ -50,4 +70,31 @@ public async ValueTask DisposeAsync()
             await _context.DisposeAsync();
         }
     }
+
+    protected async Task TakeScreenshotAsync(string path, IPage page)
+    {
+        var screenshotPath = Path.Combine(GetLogPath(), path);
+        await page.ScreenshotAsync(new PageScreenshotOptions { Path = screenshotPath });
+        _output.WriteLine($"Dashboard screenshot saved to {screenshotPath}");
+    }
+
+    private string GetLogPath()
+    {
+        var testLogPath = Environment.GetEnvironmentVariable("TEST_LOG_PATH");
```

Comment:
This is only set for helix jobs. On the build machine we want to use `artifacts/log` directory, which gets collected into `Artifacts` shown in azdo.